Packs/CortexXDR/Scripts/InvestigationVerdict/InvestigationVerdict.py

```diff
@@ -0,0 +1,28 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+RED_HTML_STYLE = "color:#FF1744;text-align:center;font-size:300%;>"
```

Comment:
As this script in the XDR pack, and the name is very generic - please add a prefix Cortex XDR - Investigation Verdict.

Reply:
Done